src/main/java/com/ontotext/kafka/service/UpdateRecordsProcessor.java

```diff
@@ -34,14 +39,23 @@ protected UpdateRecordsProcessor(Queue<Collection<SinkRecord>> sinkRecords, Atom
 	@Override
 	protected void handleRecord(SinkRecord record, RepositoryConnection connection) {
 		try {
+			LOG.trace("Executing update graph operation......");
+			long start = System.currentTimeMillis();
 			String query = getQuery(record);
 			connection.prepareUpdate(query)
-				.execute();
+					.execute();
 			connection.add(ValueUtil.convertRDFData(record.value()), format);
+			long finish = System.currentTimeMillis();
+			if (LOG.isTraceEnabled()) {
+				LOG.trace("Record info: {}", ValueUtil.recordInfo(record));
+				LOG.trace("Converted the record and added it to the RDF4J connection for {} ms", finish - start);
+			}
 		} catch (IOException e) {
+			LOG.debug("Caught an I/O exception while processing record");
```

Comment:
Not sure if those should be in debug. wdyt?

Reply:
Having in mind the above comments, I've decided to remove these logs entirely